### DISCUSSION TITLE
Introduce a new view prop for vertical tracks

### DISCUSSION THREAD

**Discussion by flekschas** [gosling-lang/gosling.js#242] [Ideas]
I wonder if we could introduce a new view prop called `orientation` to add support for vertical tracks.

This prop could either expect the following values:
- `horizontal`
- `vertical`
- `45deg`

The degree option is probably less important

**Comment by sehilyi** [comment]
I like the idea. I will add a dedicated type (i.e., `orientation: 'horizontal' | 'vertical'`) when supporting vertical tracks (https://github.com/gosling-lang/gosling.js/issues/54).

**Reply by sehilyi** [reply]
My initial thought when design Gosling was to let users map genomic coordinates with a `y` channel (e.g., `y: { field: "position", type: "genomic" }`) to make vertical tracks, but I think your `orientation` idea is better because users can easily change the orientation by using the view prop (for band connection, for example, users are using four genomic channels `x`, `xe`, `x1`, `x1e`, and changing them to `y`, `ye`, `y1`, `y1e` will be cumbersome.)

Perhaps, we need to rename `x` and `y` when using the `orientation` idea since x and y axes are determined by the `orientation` prop (currently, in `vertical` orientation `x` will be actually mapped to y-axis).
